README.md

```diff
@@ -90,13 +90,65 @@
 ## Getting Started
 
 
-### Prerequisites
-
-
-
-### Installation
-
-
+### Setup
+#### First you need is load Alpine.js and Tailwindcss
+
+```bash
+<script src="https://unpkg.com/alpinejs" defer></script>
+```
+
+```bash
+<script src="https://cdn.tailwindcss.com"></script>
+```
+
+#### Than you must add Auth Modal Library
+
+```bash 
+<script src="https://cdn.jsdelivr.net/gh/tringug/auth-modal/src/auth-modal.js" defer></script> 
+```
+
+#### And you must add Adapter SDK (You can use what you won´t)
+
```

Comment:
Tu spravne rozklikavacku, podobne ako mame na screenshoty. Vzdy ked je napisany iba nazov adaptera (zatial len Appwrite), a v tom po rozkliknuti postup ako to nainstalovat, a nakonfigurovat.